spec/tests/image/image_spec.rb

```diff
@@ -13,8 +13,12 @@
 end
 
 # when ceph is enabled, ensure we have the ceph keyring installed
-describe command("bash -c \"if grep -q 'default_store = rbd' /etc/glance/glance-api.conf; then test -f /etc/ceph/ceph.client.glance.keyring; fi\"") do
-  it { should return_exit_status 0 }
+rbd_enable = file('/etc/glance/glance-api.conf').contain("^default_store = rbd",nil,nil)
```

Comment:
@Prophidys more like `default_store\s*=\s*rbd\s*$` to allow other kinds of spaces for space-ish :rocket: people :space_invader: